src/styles/blocks/card.scss

```diff
@@ -0,0 +1,91 @@
+.card {
+  display: flex;
+  width: $card-width;
+  margin-top: 50px;
+  margin-right: 48px;
+  padding: 32px 0 16px;
```

Comment:
add paddings for left and right and you shouldn't to add paddings to each element